ibis/backends/tests/test_client.py

```diff
@@ -834,10 +885,24 @@ def test_self_join_memory_table(backend, con):
         param(
             ibis.memtable([("a", 1.0)], columns=["a", "b"]),
             id="python",
+            marks=[
+                pytest.mark.notimpl(
+                    ["flink"],
+                    raises=NotImplementedError,
+                    reason="NotImplementedError('flink') raised in repr()",
```

Comment:
totally fine to have a `reason` in here, but `reason` isn't required for a `notimpl`, the implicit reason always being "we haven't implemented it"

Reply:
Agreed; removed for all such instances. FYI @mfatihaktas